components/secondary_button/README.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Secondary buttons allow users to carry out a supporting task
+title: Secondary Button
+status: Draft
+author: orinevares
+---
+
+![Status](https://img.shields.io/badge/Recommended-Draft-orange.svg)
+> Last Updated: March 27, 2019
+
+# Secondary Button
+Secondary buttons allow users to carry out a supporting action on your service, such as Back or Cancel.
+
+## Visual Example
+
+<component-preview path="components/secondary_button/sample.html" height="100px" width="800px"> </component-preview>
+
+## Use This For
+* A supporting action you want users to take on your site.
+
+## Don't Use This For
+*	Primary actions on your page such as 'Submit', or 'Start'. Use a [Primary Button](https://developer.gov.bc.ca/Design-System/Primary-Button) instead.
+
+## Design Guidance
+*	Use for supporting actions or options you don't want users to default to
+*	Button labels should be as short as possible with action words that clearly explain what will happen when the button is clicked (eg. Back, Cancel, or Don't Agree)
+* Do not use images for buttons
+
+## Rationale
+The rounded edges on the button help differentiate it from other layout items on a page. The underline on hover ensures colour is not the only visual means of conveying information about the button being clickable. The transparent background helps the button reduce its visual prominance amongst other elements and primary buttons on a page.
+
+## Behaviour
+1. Button label should appear underlined on hover
+2. Button text and border colour should revert to #003366 when in the active state
```

Comment:
It seems that "label" and "text" are used interchangeably here. When you say active state, do you mean the default state?

Reply:
the active state is when the user clicks on the button. The sequence would go default->hover>(click)>active while clicking.